packages/babel/src/index.js

```diff
@@ -31,11 +31,19 @@ const unpackOptions = ({
 
 const unpackInputPluginOptions = ({ skipPreflightCheck = false, ...rest }, rollupVersion) => {
   if (!rest.babelHelpers) {
-    // eslint-disable-next-line no-console
-    console.warn(
-      "babelHelpers: 'bundled' option was used by default. It is recommended to configure this option explicitly, read more here: " +
-        'https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers'
-    );
+    if ('runtimeHelpers' in rest) {
+      // eslint-disable-next-line no-console
+      console.warn(
```

Comment:
We can give a concrete recommendation on how the code should be adjusted, it's just a matter of changing `runtimeHelpers: true` to `babelHelpers: 'runtime'`. The link to the docs is still nice - but should just act as "learn more" kind of thing.

Reply:
Also - the same warning should be introduced for `externalHelpers: true` which is equivalent to `babelHelpers: 'external'`

Reply:
Yeah, I'm unsure of the mapping.

As I understood it, for me, `babelHelpers: true` was [about inlining dependencies](https://github.com/renoirb/experiments-201908-rush-typescript-just-bili-monorepo/blob/20200303-version-policies/conventions/use-bili/src/plugins/babel.ts#L37-L72).

So, at least a "learn more" is better than a warning about something I didn't know changed is good.

Reply:
Oh, the home page has *bit rot*, maybe we should fix that too.

<img width="832" alt="Screen Shot 2020-08-10 at 6 55 49 PM" src="https://user-images.githubusercontent.com/296940/89839211-395a6400-db3b-11ea-8930-b441224df5a1.png">

Reply:
Oh. I see.
*externalHelpers* and *runtimeHelpers* are mutually exclusive and now instead of two attributes with boolean, they're now as a string under *babelHelpers*.

Reply:
Yes, that's basically the idea behind those - the meaning of them hasn't changed much, it's only a couple of booleans converted to a single enum.